src/typing/type_inference_js.ml

```diff
@@ -5084,6 +5113,8 @@ and mk_class_elements cx instance_info static_info body = Ast.Class.(
       let this, super, method_sigs, getter_sigs, setter_sigs =
         if static then static_info else instance_info
       in
+      let yield = MixedT (mk_reason "no yield" loc) in
```

Comment:
I... actually didn't realize generator methods were possible. I'll add a suite of tests and add the probably missing implementation stuff. This PR should be blocked until then. Thanks for catching this.